graphql_client_codegen/src/shared.rs

```diff
@@ -77,7 +82,13 @@ pub(crate) fn field_impls_for_selection(
                 let ty = fields
                     .iter()
                     .find(|f| &f.name == name)
-                    .ok_or_else(|| format_err!("could not find field `{}`", name))?
+                    .ok_or_else(|| {
+                        format_err!(
+                            "Could not find field `{}`. Available fields: `{}`.",
+                            name,
+                            format_available_fields(fields).as_str()
```

Comment:
This `as_str` seems superfluous now (same below).

Reply:
Right - thanks!